reference/5.1/PSReadline/About/about_PSReadline.md

```diff
@@ -0,0 +1,959 @@
+Does not cancel input like CancelLine.
+
+CharacterSearch
+
+- Cmd: `<F3>`
+- Emacs: `<Ctrl+]>`
+
+Read a key and search forwards for that character. With an argument, search
+forwards for the nth occurrence of that argument. With a negative argument,
+searches backwards.
+
+### CharacterSearchBackward
+
+- Cmd: `<Shift+F3>`
+- Emacs: `<Alt+Ctrl+]>`
+
+Like CharacterSearch, but searches backwards. With a negative argument,
+searches forwards.
+
+### ClearScreen
+
+- Cmd: `<Ctrl+L>`
+- Emacs: `<Ctrl+L>`
+
+Clears the screen and displays the current prompt and input at the top of the
+screen.
+
+### DigitArgument
+
+- Cmd: unbound
+- Emacs: `<Alt+[0..9]>`,,`<Alt+->`
+
+Used to pass numeric arguments to functions like CharacterSearch or
+YankNthArg. Alt+- toggles the argument to be negative/non-negative. To enter
+80 '*' characters, you could type Alt+8 Alt+0 *.
+
+CaptureScreen
+
+- Cmd: unbound
+- Emacs: unbound
+
+Copies selected lines to the clipboard in both text and RTF formats. Use
+up/down arrow keys to the first line to select, then
+Shift+UpArrow/Shift+DownArrow to select multiple lines. After selecting, press
+Enter to copy the text. Escape/Ctrl+C/Ctrl+G cancel the operation, so nothing
+is copied to the clipboard.
+
+### InvokePrompt
+
+- Cmd: unbound
+- Emacs: unbound
+
+Erases the current prompt and calls the prompt function to redisplay the
+prompt. Useful for custom key handlers that change state, such as changing the
+current directory.
+
+### WhatIsKey
+
+- Cmd: `<Alt+?>`
+- Emacs: `<Alt+?>`
+
+Read a key or chord and display the key binding.
+
+### ShowKeyBindings
+
+- Cmd: `<Ctrl+Alt+?>`
+- Emacs: `<Ctrl+Alt+?>`
+
+Show all of the currently-bound keys.
+
+### ScrollDisplayUp
+
+- Cmd: `<PageUp>`
+- Emacs: `<PageUp>`
+
+Scroll the display up one screen.
+
+### ScrollDisplayUpLine
+
+- Cmd: `<Ctrl+PageUp>`
+- Emacs: `<Ctrl+PageUp>`
+
+Scroll the display up one line.
+
+### ScrollDisplayDown
+
+- Cmd: `<PageDown>`
+- Emacs: `<PageDown>`
+
+Scroll the display down one screen.
+
+### ScrollDisplayDownLine
+
+- Cmd: `<Ctrl+PageDown>`
+- Emacs: `<Ctrl+PageDown>`
+
+Scroll the display down one line.
+
+### ScrollDisplayTop
+
+- Cmd: unbound
+- Emacs: `<Ctrl+Home>`
+
+Scroll the display to the top.
+
+### ScrollDisplayToCursor
+
+- Cmd: unbound
+- Emacs: `<Ctrl+End>`
+
+Scroll the display to the cursor.
+
+## Custom Key Bindings
+
+PSReadLine supports custom key bindings using the cmdlet
+`Set-PSReadLineKeyHandler`. Most custom key bindings call one of the above
+functions, for example
+
+```powershell
+Set-PSReadLineKeyHandler -Key UpArrow -Function HistorySearchBackward
+```
+
+You can bind a ScriptBlock to a key. The ScriptBlock can do pretty much
+anything you want. Some useful examples include
+
+- edit the command line
+- opening a new window (e.g. help)
+- change directories without changing the command line
+
+The ScriptBlock receives two arguments:
+
+- `$key` - A `[ConsoleKeyInfo]` object that is the key that triggered the
```

Comment:
Should the object be codefenced or just bold?

Reply:
I always forget which way. Style guide says bold.